lib/utils/JsonHelper.dart

```diff
@@ -84,7 +84,7 @@ class JsonHelper {
           JsonObject.parseInt(fieldObject['angle']),
         );
         final String? coordinatesImageSize =
-            fieldObject['coordinates_image_size'];
+            fieldObject['coordinates_image_size'].toString();
```

Comment:
Should be `fieldObject['coordinates_image_size']?.toString();`
For some reason that I don't understand, when `fieldObject['coordinates_image_size'] == null`, `fieldObject['coordinates_image_size'].toString()` returns the `String` `'null'` instead of crashing - and we expect the `null` value instead.